packages/eslint-plugin/docs/rules/class-methods-use-this.mdx

```diff
@@ -33,8 +33,6 @@ const defaultOptions: Options = {
 
 {/* insert option description */}
 
-Makes the rule ignore any class member explicitly marked with `override`.
```

Comment:
Nice! I just noticed the resulting doc page had more-or-less the same option description twice, as the `/* insert option description */` comments add the original option description.

I've checked if there are other rules that have this issue which led me to #10748. Some rules have options with the same description (for example: https://typescript-eslint.io/rules/no-confusing-void-expression/#ignorearrowshorthand).

I wonder if/how this can be prevented.

Reply:
Yeah we could probably add a lint rule or test to check that there isn't a strict duplicate. Or loose duplication if we remove spaces, lowercase, or even check something like the Levenshtein distance and/or [`are-docs-informative`](https://www.npmjs.com/package/are-docs-informative)-style distance. But there are so many that are very differently phrased -and we aren't adding new violations- I don't think it'd catch much long-term. +1 to #10748 from me as a one-time thing.